csharp.html

```diff
@@ -4,12 +4,13 @@
 //Common testing requirement. If you are consuming an API in a sandbox/test region, uncomment this line of code ONLY for non production uses.
 //System.Net.ServicePointManager.ServerCertificateValidationCallback = delegate { return true; };
 
-//Be sure to run "Install-Package Microsoft.AspNet.WebApi.Client" from your nuget command line.
+//Be sure to run "Install-Package Microsoft.Net.Http" from your nuget command line.
+using System;
 using System.Net.Http;
 
-var baseAddress = "<%- @apiUrl %>";
+var baseAddress = new Uri("<%- @apiUrl + "/" %>");
```

Comment:
You probably don't want to add the slash, because then you're definitely adding `/` to any URLs in the future.

What about routes of these types: `http://example.com?w=1` or `http://example.com:8080?soap=me` ?